packages/grid/x-data-grid-generator/src/useMovieData.ts

```diff
@@ -0,0 +1,284 @@
+import { GridColumns, GridComponentProps, GridRowModel } from '@mui/x-data-grid';
+
+type Movie = {
+  title: string;
+  gross: number;
+  director: string;
+  company: string;
+  year: number;
+  composer: { name: string };
+  cinematicUniverse?: string;
+};
+
+const COLUMNS: GridColumns = [
+  { field: 'title', headerName: 'Title', width: 200, groupable: false },
```

Comment:
Could this new dataset be an option of `useDemoData` like "commodities" and "employees"? I mean to have a single entry point to get sample data.

Reply:
I moved it to another entry point to differentiate auto-generated data and fine-tuned datasets written by hand.